jenkins-master/jobs/mozilla-aurora_functional/config.xml

```diff
@@ -89,24 +89,33 @@
       <selector class="hudson.plugins.copyartifact.StatusBuildSelector"/>
     </hudson.plugins.copyartifact.CopyArtifact>
     <hudson.plugins.xshell.XShellBuilder plugin="xshell@0.9-SNAPSHOT">
-      <commandLine>python runtests.py --type=functional --branch=mozilla-aurora --build-revision=$REVISION --build-locale=$LOCALE --installer-url=$INSTALLER_URL</commandLine>
+      <commandLine>python -u submission.py --test-type=functional --state=running --repository=mozilla-aurora --revision=$REVISION --locale=$LOCALE treeherder_venv</commandLine>
```

Comment:
As you will see in the script we dynamically build-up the necessary treeherder data, also by specifying settings for each kind of test. For media tests we most likely will use `--type=media` later.